src/SimpleNotePad.java

```diff
@@ -0,0 +1,181 @@
+import javax.swing.JFrame;
+import javax.swing.JMenu;
+import javax.swing.JMenuBar;
+import javax.swing.JMenuItem;
+import javax.swing.JOptionPane;
+import javax.swing.JScrollPane;
+import javax.swing.JTextPane;
+import javax.swing.text.BadLocationException;
+import javax.swing.text.Position;
+import javax.swing.text.SimpleAttributeSet;
+import javax.swing.text.StyledDocument;
+
+public class SimpleNotePad extends JFrame implements ActionListener {
+	// global variables
+	private JMenuBar menuBar;
+	private JMenu fileMenu;
+	private JMenu editMenu;
+	private JMenu recentFile;
+	private JTextPane notePad = new JTextPane();
+	private JMenuItem newFile;
+	private JMenuItem saveFile;
+	private JMenuItem printFile;
+	private JMenuItem openFile;
+	private JMenuItem copy;
+	private JMenuItem paste;
+	private JMenuItem replace;
+	private JMenuItem[] recent;
+	private Integer count;
+
+	// constructor
+	public SimpleNotePad() {
+		instantiateMenuObjects(); // instantiates the menu objects
+		addMenuItems(); // adds menu items to menubar
+		configureMenuBars();
+		configureActionListeners();
+		configureJFrameSettings();
+	}
+
+	// instantiates the menu objects
+	private void instantiateMenuObjects() {
+		menuBar = new JMenuBar();
+		fileMenu = new JMenu("File");
+		editMenu = new JMenu("Edit");
+		recentFile = new JMenu("Recent");
+		notePad = new JTextPane();
+		newFile = new JMenuItem("New File");
+		saveFile = new JMenuItem("Save File");
+		printFile = new JMenuItem("Print File");
+		openFile = new JMenuItem("Open File");
+		copy = new JMenuItem("Copy");
+		paste = new JMenuItem("Paste");
+		replace = new JMenuItem("Replace");
+		recent = new JMenuItem[3];
+
+		count = 0;
+	}
+
+	// configures the JFrame settings
+	private void configureJFrameSettings() {
+		setTitle("A Simple Notepad Tool");
+		add(new JScrollPane(notePad));
+		setPreferredSize(new Dimension(600, 600));
+		setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+		setVisible(true);
+		pack();
+
+	}
+
+	// adds actions listeners
+	private void configureActionListeners() {
+		setActionListeners(newFile, "new");
+		setActionListeners(saveFile, "save");
+		setActionListeners(printFile, "print");
+		setActionListeners(openFile, "open");
+		setActionListeners(copy, "copy");
+		setActionListeners(paste, "paste");
+		setActionListeners(replace, "replace");
+
+	}
+
+	// Sets the action listener and commands based on item and command text
+	private void setActionListeners(JMenuItem item, String command) {
+
+		item.addActionListener(this);
+		item.setActionCommand(command);
+	}
+
+	// adds menu items
+	private void addMenuItems() {
+
+		addFileMenuItems();
+		addEditMenuItems();
+
+	}
+
+	// adds file menu items
+	private void addFileMenuItems() {
+		fileMenu.add(newFile);
+		fileMenu.addSeparator();
+		fileMenu.add(saveFile);
+		fileMenu.addSeparator();
+		fileMenu.add(printFile);
+		fileMenu.addSeparator();
+		fileMenu.add(openFile);
+		fileMenu.addSeparator();
+		fileMenu.add(recentFile);
+
```

Comment:
To increase code readability and reduce code duplication, we could try using putting the JMenuItems into a Collection such as an ArrayList and iterate through that array. 
```suggestion
ArrayList<JMenuItem> fileMenuItems = new ArrayList<JMenuItem>(Arrays.asList(newFile, saveFile, printFile, openFile, recentFile));
    	for(JMenuItem menuItem : fileMenuItems){
    		fileMenu.add(menuItem);
    		fileMenu.addSeparator();
    	}
```